NEWS.md

```diff
@@ -67,6 +68,16 @@ Note: We're only listing outstanding class updates.
     * Module#set_temporary_name added for setting a temporary name for a
       module. [[Feature #19521]]
 
+* NoMethodError
```

Comment:
I don't think this change is worth mentioning in NEWS.

Reply:
I think it is better to explain this in "Compatibility issues" section than in "Core classes updates".

Reply:
@jeremyevans My thinking here is this:
1. It is a part of public behavior
2. It is a behavior that is frequently encountered (i.e., almost every Rubyist sees this error almost every day, while some other notable changes like `WeakKeyMap` or `Fiber#kill` are something that wouldn't be used by as many people)
3. The change affects the quality of life. I understand the justification, yet the change might make somebody's debugging harder (by not seeing the real problematic object) and somebody's easier (by removing 20-lines `#inspect`s of complicated objects)
4. The change also might affect application/library code (this is purely theoretical, but not unimaginable that some code caught the error and extracted the `#inspect` of the affected object; or that somebody's `#inspect` was defined in the way that considered frequent rendering in this message)
5. The entry in `NEWS` not only explains the change but links to the corresponding discussion, so when one sees in the new version, "wait, is this changed?" they can a) find the entry in NEWS (and make sure the change was conscious, not some side-effect or bug) and b) find the justification.

Reply:
@zverok I read what you wrote and still do not think this is worth mentioning.  There is a tradeoff in mentioning anything in NEWS.  We want NEWS to be short enough that people will read it, hitting the important parts of the release.  NEWS is not designed to be comprehensive and discuss every single change since the last release. If you think this is a compatibility issue (I do not), take @mame's advice and put it in the compatibility issues section. Otherwise, I think it should be removed.

Reply:
I'm for because it is a part of public behavior.
Generally speaking NEWS.md is allowed to write about something if a committer thinks it should be.
I think what jeremy says  is rather a release note.